evm/src/main/java/com/horizen/evm/StateDB.java

```diff
@@ -10,6 +10,12 @@ public class StateDB extends ResourceHandle {
     public static byte[] EMPTY_CODE_HASH =
             Converter.fromHexString("c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470");
 
+    /**
+     * TrieHasher.Root() of an empty byte array
+     */
+    public static byte[] EMPTY_ROOT_HASH =
+            Converter.fromHexString("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421");
```

Comment:
Aren't Converter.fromHexString/toHexString the same as BytesUtils.fromHexString/toHexString?

Reply:
Yes, they are, and there is another similar util in package Numeric. I used this one for being consistent with the overall usage in this package, but probably we could live with one- util package only.

Reply:
ok